hummingbot/connector/derivative/binance_perpetual/binance_perpetual_derivative.py

```diff
@@ -259,7 +259,7 @@ async def create_order(self,
                                               add_timestamp = True,
                                               is_signed=True)
             exchange_order_id = str(order_result["orderId"])
-            tracked_order = self._in_flight_orders.get(order_id)
+            tracked_order = self._in_flight_orders.get(order_id, None)
```

Comment:
The default `None` does not make much sense since in line 253 we call `start_tracking_order` that includes the new order in the inflight orders list.